Framework/Core/include/Framework/AnalysisDataModel.h

```diff
@@ -438,17 +437,22 @@ DECLARE_SOA_EXTENDED_TABLE(MFTTracks, StoredMFTTracks, "MFTTRACK", //!
 
 using MFTTrack = MFTTracks::iterator;
 
+namespace fwdtrack // Index to MFTtrack column must be defined after table definition. Should a SELF_INDEX_COLUMN be used here?
+{
+DECLARE_SOA_INDEX_COLUMN(MFTTrack, matchMFTTrack); //! ID of matching MFT track for GlobalMuonTracks
+}
```

Comment:
@jgrosseo , notice here that in order to define the index to MFT track column, the corresponding mfttrack table must be already defined. This breaks the column definitions in two parts.

Reply:
The index is right and also your observation. You can remove "Should a SELF_INDEX_COLUMN be used here?"